lua/lint/linters/checkov.lua

```diff
@@ -0,0 +1,54 @@
+local severities = {
+  error = vim.diagnostic.severity.ERROR,
+  warning = vim.diagnostic.severity.WARN,
+  refactor = vim.diagnostic.severity.INFO,
+  convention = vim.diagnostic.severity.HINT,
+}
+
+return {
+  cmd = 'checkov',
+  stdin = true,
+  args = {
+    '-f',
+    vim.api.nvim_buf_get_name(0),
```

Comment:
```suggestion
    function() return vim.api.nvim_buf_get_name(0) end,
```

I think otherwise this is only evaluated once when the linter is initially loaded via a `require` call. Subsequent usages would then always re-use the filename of the file that got linted first within a neovim session.


Also - are you sure it is right to provide both `-f` and also set `stdin = true` ?